apps/daimo-mobile/src/action/useSendAsync.ts

```diff
@@ -126,14 +126,19 @@ async function sendAsync(
     return handle;
   } catch (e: any) {
     console.error(e);
-    if (keySlot === undefined) setAS("error", "Device removed from account");
-    else setAS("error", "Error sending transaction");
+    if (keySlot === undefined) {
+      setAS("error", "Device removed from account");
+    } else if (e instanceof NamedError && e.name === "ExpoEnclaveSign") {
+      setAS("error", e.message);
```

Comment:
so this passes thru `Passcode not set` etc

Reply:
yep